docs/recipe_overview.md

```diff
@@ -0,0 +1,440 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Recipes: An Overview
+
+By default, Kubernetes resources are Captured and Recovered as single, namespaced
+groups. Some applications may require strict ordering by type of resource during
+Capture or Recover to ensure a successful deployment. To provide this flexibility,
+Recipes can be used to describe a Workflow used for a Capture or Recover action.
+These Recipe Workflows can be referenced by the VolumeReplicationGroup (VRG), and
+executed periodically.
+
+Recipe source code and additional information can be found [here](https://github.com/RamenDR/recipe).
+The current Recipe CRD can be found [here](https://github.com/RamenDR/recipe/blob/main/config/crd/bases/ramendr.openshift.io_recipes.yaml).
+
+## Overview of this document
+
+1. What is a Recipe and why do we need it?
+1. Recipe concepts
+1. Sample application that uses a Recipe
+1. Additional details on Recipe functionality
+1. Where to go to learn more
```

Comment:
(nit) Section links would be nice
https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links